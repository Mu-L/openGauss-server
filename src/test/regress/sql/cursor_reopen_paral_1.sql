start transaction;
call proc1();
select pg_sleep(2);
commit;

start transaction;
call proc2();
select pg_sleep(1);
call proc1();
commit;