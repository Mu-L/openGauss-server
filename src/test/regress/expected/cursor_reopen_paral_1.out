start transaction;
call proc1();
INFO:  v_lastname is wang
INFO:  v_jobid is 1
INFO:  ----------------------------
INFO:  v_age is 1
INFO:  v_depid is 1
INFO:  ----------------------------
 proc1 
-------
 
(1 row)

select pg_sleep(2);
 pg_sleep 
----------
 
(1 row)

commit;
start transaction;
call proc2();
INFO:  v_lastname is wang
INFO:  v_jobid is 1
INFO:  ----------------------------
INFO:  v_age is 1
INFO:  v_depid is 1
INFO:  ----------------------------
 proc2 
-------
 
(1 row)

select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

call proc1();
INFO:  v_lastname is wang
INFO:  v_jobid is 1
INFO:  ----------------------------
INFO:  v_age is 1
INFO:  v_depid is 1
INFO:  ----------------------------
 proc1 
-------
 
(1 row)

commit;
